Ошибка: HTTPConnectionPool(host='78', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x00000202765434C0>: Failed to resolve '78' ([Errno 11001] getaddrinfo failed)"))